src/domain/flow-system/src/entities/shared/schedule.rs

```diff
@@ -57,24 +95,31 @@ pub struct CronExpressionIterationError {
 /////////////////////////////////////////////////////////////////////////////////////////
 
 impl Schedule {
-    pub fn validate_cron_expression(
-        cron_expression: String,
-    ) -> Result<CronSchedule, ScheduleError> {
-        let schedule = match CronSchedule::from_str(&cron_expression) {
+    pub fn new_cron_schedule(source: CronExpressionSource) -> Result<Schedule, ScheduleError> {
+        let cron_as_string_with_dummy_seconds = format!(
```

Comment:
Just a proposal, for me looks useful to extend `CronExpressinSource` with `as_cron_string()` method

Reply:
Made a method, might be useful